### ISSUE TITLE
[Adaptive Card iOS - InputLabelPosition.json]: Screen Reader is announcing incorrect role as 'Button' for the 'Select Color'.

### ISSUE THREAD

**Issue by vagpt** [microsoft/Teams-AdaptiveCards-Mobile#174]
### Target Platforms

iOS

### SDK Version

Version 1.0 (2.4.0-beta.24.5.30.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 17.5.1
Application: Adaptive Cards
App Version: Version 1.0 (2.4.0-beta.24.5.30.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Navigate to "V1.6" control and select it.
4. Navigate to 'Elements' control and activate it.
5. Navigate to 'InputLabelPosition.json' under 'Elements' list item and activate it.
6. Card will be appear.
7. Perform right swipe, navigate to the 'Select Color' combo box and  observe the issue.

**Actual Result:**
Incorrect role as 'Button' is defined for the select color combo box. When the voice over focus lands on it then screen reader is announcing as 'Select color Red button.'

**Expected Result:**
Screen Reader should announce the correct role as 'Combo box' for the 'Select color'. It should announce as **'Select color Red combo box collapsed double tap to activate it.'**

**Note:**
Same issue is repro throughout the application for the same control type.

**User Impact:**
It will impact the screen reader user as they will not be able to know whether the control is expandable or not if the screen reader will announce the role as button only.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section.

### Screenshots

https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/ffb2f01b-b4a5-4a17-8fe0-1cf4737f9eb5

### Card JSON

```JSON
NA
```

**Comment by ardlank** [comment]
@hggz take a look
